### ISSUE TITLE
[missing stuff? I really never know what to put here][2.0.8] not reading drawer controller

### ISSUE THREAD

**Issue by winsrp** [MrRiegel/Storage-Network#121]
Using storage drawers it will pick individual/basic drawers and the compacting drawer correctly but when linked to the drawer controller or drawer slave, nothing is picked up.

**Comment by KidsDontPlay** [comment]
works well for me. do hoppers work with controller/slave?

**Comment by winsrp** [comment]
yes they do, I was kind of curious so I just tested it, putting items with a hopper on a controller and slave, input the items to the correct drawer, and if and item is not in a drawer and there is an empty drawer it will start filling that one, extracting from the drawer with a hopper seems to start with the drawer which is closer to the controller and then keep going from there.
